### DISCUSSION TITLE
What powers the blog?

### DISCUSSION THREAD

**Discussion by trusktr** [pulsar-edit/.github#188] [Documentation & Website]
Hello! The blog is nice and clean. What stack is powering it?

**Comment by trusktr** [comment]
Ah, I see, VuePress and [theme-hope](https://theme-hope.vuejs.press/) theme. Nice customization!

**Reply by Daeraxa** [reply]
I agree, it looks nice but we are running into issues for it for our main docs (not so much the blog) so we will likely be moving away from it bit by bit. No plans for the blog to go anywhere just yet though :P
